test/test_git.py

```diff
@@ -29,3 +30,26 @@ def test_repo_owner(self):
 
     def test_repo_name(self):
         assert_equal('apiary-test', self.parsed['name'])
+
+
+class TestGitBranchOutput(object):
+
+    @patch('subprocess.check_output')
+    def test_branch_name_parsing(self, check_output_mock):
+        branch_name = 'prefix/branch-name'
```

Comment:
This could be simplified by using `b'prefix/branch-name'` since this would work on both Pythons, on Python 3 it would be bytes and 2 a string.

```python
check_output_mock.return_value = b'prefix/branch-name\n'
assert_equal('prefix/branch-name', get_current_branch())
```